docs/system/wires_fuses.md

```diff
@@ -26,40 +26,85 @@ Specific resistance is the property of the material used for the wire. Different
 
 <figure>
     <wire-efficiency/>
     <br/>
 <center>
-    <figcaption><b>Figure 1.</b> Efficiency and voltage drop variations of electrical wires <b>(interactive)</b>.</figcaption>
+    <figcaption><b>Figure 1.</b> Efficiency variations of electrical wires <b>(interactive)</b>.</figcaption>
 </center>
 </figure>
 
+The voltage drop across the wire has to be considered when the [charge controller](charge_controller) is configured to maintain the proper voltage on the solar panels.
+
+<figure>
+    <wire-voltage-drop/>
+    <br/>
+<center>
+    <figcaption><b>Figure 1.</b> Voltage drop across 20m long wires with different cross section area.</figcaption>
+</center>
+</figure>
+
+### Example
+
+The distance between the panels and the charge-controller shell be $D=5m$, which means a total length (two directions) of $l_{wire} = 10m$. For a copperwire with a diameter of $d=1.5mm$, the total resistance of the wire (for an ambient temperature of $30^{\circ}C$) is
+
+$$R=\frac{0.0172 \cdot 10}{1.5} \Omega = 0.114 \Omega$$
+
+If we assume a maximum current of $I_{max} = 10A$ like charge controller [MPPT 1210 HUS](https://libre.solar/hardware/mppt-1210-hus.html) can handle, the maximum power dissipation in the wire is
+
+$$P_{wire} = I_{max}^2 * R = 100A^2 * 0.114 \Omega = 11.4W$$
+
+In most cases, this approach is rather used to determine the minimal cross section for the wires. If a power loss of 2% of the peak power is tolerated and the maximum expected temperature in the wire is assumed to be $50^{\circ}C$, the cross section can be determined using the above formulas. Let the peak power be $P_{peak}=120W$. The specific resistance at $50^{\circ}C$ is $\rho_{50} = 0.0185\frac{\Omega mm^2}{m}$.
+
+$$P_{wire} = 2\%\cdot P_{peak} = 2.4W$$
+$$R_{wire} = \frac{P_{wire}}{I_{max}^2} = \frac{2.4W}{100A} = 0.024\Omega$$
+$$A_{wire} = \frac{\rho \cdot l_{wire}}{R_{wire}} = 7.7mm^2$$
+
+So for this example, a wire with a cross section of $A=7.7mm^2$ should suffice.
+
+For short connections below 5 meters, you can use:
+
+::: warning Rule of Thumb:
+ current / 3 = cable cross section area
+:::
+
+Table 2 lists common cross section for some situations. Note that the length is given as the total of both positive and negative lines.
+
+|                           | $length=5m$        |  $length=10m$|
+|:-------------------------:|:------------------:|:------------------:|
+|Cross section in $[mm^2]$  | max. current $[A]$ |  max. current $[A]$|
+|   0.75        		    |        2.3         |      1.1
+|   1.5        		        |        4.5         |      2.3
+|   10        		        |        30          |      15
+|   35        		        |        105         |      53
+|   50        		        |        150         |      75
+|   120        		        |        360         |      180
```

Comment:
Can you use spaces for indentation? This looks chaotic.

Reply:
I used spaces, I guess there were indentations already in there. Will fix it